app/controllers/aid_applications/verifications_controller.rb

```diff
@@ -7,7 +7,7 @@ def edit
     def update
       @aid_application = current_aid_application
       @aid_application.assign_attributes(aid_application_params)
-      @aid_application.save(context: :verification)
+      @aid_application.save(context: :submit)
```

Comment:
Make a new context for just these page that just includes validations for fields on this page

Reply:
Done